region/lowernorfair/west/Fast Ripper Room.json

```diff
@@ -98,6 +98,103 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          {"and": [
+            "ScrewAttack",
+            {"heatFrames": 350}
+          ]},
+          {"and": [
+            {"enemyKill": {"enemies": [["Ripper 2 (red)", "Ripper 2 (red)"]], "explicitWeapons": ["PowerBomb"]}},
+            {"heatFrames": 450}
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            {"or": [
+              "ScrewAttack",
+              {"enemyKill": {"enemies": [["Ripper 2 (red)"]], "explicitWeapons": ["PowerBomb"]}}
+            ]},
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 120}}
```

Comment:
The order of events either has you kill the rippers early, and disable the etanks before taking heatFrames.  Or have a harder time crossing the room, requiring more tech or enemyDamage.